### PR TITLE
travis: speed up travis

### PR DESCRIPTION
**Pull request by markwolff**
Parallelize travis jobs

### CODE REVIEW

File: .travis/test_react_native.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+cd extensions/applicationinsights-react-native
```
Comment (xiao-lix) [inline .travis/test_react_native.sh]: what about adding `test_angular.sh` like so: 
`
cd extensions/applicationinsights-angular
npm install
npm run build
npm run test
npm run lint
`
we can also adding it later after angular plugin is published.

Comment (markwolff) [reply .travis/test_react_native.sh]: added `test_angular.sh`

File: .travis.yml [hunk 2]
```diff
@@ -6,50 +6,7 @@ before_install:
...
-    #  script:
-    #  - cd extensions/applicationinsights-react-native
-    #  - npm install
-    #  - npm run build
-    #  - npm run test
-    #  - npm run lint
+    - script: ./.travis/test_rush.sh
+      name: "Tests"
+    - script: ./.travis/test_legacy.sh
+      name: "Legacy Tests"
```
Comment (hectorhdzg) [inline .travis.yml]: Why react and angular are not running?

Comment (markwolff) [reply .travis.yml]: React Native & Angular tests aren't including in CI as of yet. React runs as part of `test_rush`.

Comment (markwolff) [reply .travis.yml]: Added react to CI. @xiao-lix can add Angular in separate PR once it is ready

**Pull request merged**